packages/mobile/src/screens/personal-info/components/profile-image/profile-image.tsx

```diff
@@ -1,31 +1,104 @@
-import React, { FC } from 'react';
+/* eslint-disable no-extra-boolean-cast */
```

Comment:
It's a bad practice to disable linter rules on entire file

Reply:
updated